internal_handlers.go

```diff
@@ -42,9 +43,17 @@ func InternalSourceList(c echo.Context) error {
 		return err
 	}
 
+	// Skip Sources which do not have associated applications or RHC Connections. Useful for the Sources Monitor since
+	// it prevents returning Sources that do not need to have availability checks performed for them. More information
+	// here: https://issues.redhat.com/browse/RHCLOUD-38735.
+	var skipEmptySources = false
+	if skip := c.Request().Header.Get(headers.SkipEmptySources); skip != "" {
+		skipEmptySources = skip == "true"
```

Comment:
Might be good to call `strings.ToLower()` on skip?

Reply:
Sounds good!